### ISSUE TITLE
Youtube "Music in this video" section

### ISSUE THREAD

**Issue by fishstik** [web-scrobbler/web-scrobbler#1904]
Some videos that feature songs are now being 'tagged' with a **Music in this video** section in the description. It contains a consistently-formatted place to retrieve Song/Artist/Album info.

Example where this would be useful (currently incorrectly named based on video title): https://www.youtube.com/watch?v=ogy5JVextNA
![image](https://user-images.githubusercontent.com/4278123/52436318-010b3d00-2ae2-11e9-9e69-de05a604a82a.png)

Note that this section can contain multiple songs. If so, ignoring or just scrobbling the first listing could be implemented. #

**Comment by stephanebruckert** [comment]
I doubt there is an easy solution for this. What to do when there is more than 1 track? for example in a set https://www.youtube.com/watch?v=jTBP3yGzJqE

Inversely, scrobbling songs "Provided to YouTube by" looks possible as they seem to always refer to a single song, see https://github.com/web-scrobbler/web-scrobbler/issues/1896

**Comment by alexesprit** [comment]
I'm not sure it should be fixed/resolved. As @stephanebruckert said it's not possible to get start time for tracks listed in this section.

**Comment by just1co** [comment]
What about "music in this video" with playlist videos that contain chapters like this?

![image](https://user-images.githubusercontent.com/34866475/119833521-d8541f00-bf07-11eb-8086-45be03b3e5d5.png)
![image](https://user-images.githubusercontent.com/34866475/119833569-e2761d80-bf07-11eb-9039-0281a44d9dd7.png)

(video: https://www.youtube.com/watch?v=Q0uKgqAjEx4)

I suppose it would not be that hard to parse these
